evals/eval.py

```diff
@@ -162,7 +163,31 @@ def worker_thread(args):
     def get_samples(self):
         if self.samples_jsonl is None:
             raise ValueError(
-                    "To use `get_samples`, you must provide a `samples_jsonl` path."
-                    "Got `None`.")
+                "To use `get_samples`, you must provide a `samples_jsonl` path." "Got `None`."
+            )
+
+        if os.path.exists(self.samples_jsonl):
```

Comment:
`self.samples_jsonl` is a relative path right? e.g. 'test_fuzzy_match/samples.jsonl'. So we should prepend the current dir

Reply:
yep good catch, fixed this